server/src/main/java/com/broadcom/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -291,7 +288,7 @@ public Class visitLocalStorageSection(LocalStorageSectionContext ctx) {
 
   @Override
   public Class visitStatement(StatementContext ctx) {
-    List<Token> tokenList =
+    @NonNull List<Token> tokenList =
```

Comment:
<img width="740" alt="CobolVisitior_1" src="https://user-images.githubusercontent.com/10369965/98957034-07bdb400-2501-11eb-86b8-8232886ea9a1.png">

Reply:
Sorry, but it will not work this way. Lombok's NonNull works only with parameters.